Details on the Online Program for SEA-ISS 2025

negin513 opened the pull request.

negin513 committed:
update blurbs about 2025 notebook proceeding office hours

negin513 committed:
adding keynote a -- confirmed

negin513 committed:
italic for speaker

vanderwb reviewed (changes requested):
Looks good - thanks for doing this. Only requested change is to fill in the office hours description for Monday as well. This seems to only change Tues and Thurs entries.

kafitzgerald reviewed (commented):
Looks good to me!  (w/ the exception of the concern Brian raised)

negin513 committed:
adding office hours for day 1

negin513 committed:
adding Ligia

negin513 committed:
adding invited

negin513 committed:
Merge branch 'main' into iss-2025-updates

negin513 committed:
tba for consistency

negin513 commented:
@vanderwb Thanks for the review. I updated this and made a few other minor changes. I also added Ligia as our keynote. Please feel free to review and merge this if it looks good.

vanderwb reviewed (approved):
_LGTM_

vanderwb commented:
Thanks for getting these changes done!
